co-log-core/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ Change log
 `co-log-core` uses [PVP Versioning][1].
 The change log is available [on GitHub][2].
 
+0.0.0.2
```

Comment:
This should also be `0.1.0` to match what we have in the `.cabal` (the real version 🙂)
Also, as `0.0.0.1` is not released you can put all the changes from there (which are in the below section) in `0.1.0` as well.